media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="
+dav1d is a new AV1 cross-platform decoder, open-source, and focused on speed and correctness. \
+It supports all features from AV1, including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+SOURCE_DIR="dav1d-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
SECONDARY_ARCHITECTURES is missing. I doubt it actually builds on x86_gcc2.

Reply:
I've seen that some recipes does not have this option.
What does this option do?

I see it very often in `x86`-only recipes, but dav1d says that it supports many other architectures, such as `arm`.

Reply:
If it doesn't build for gcc2 you need to add SECONDARY_ARCHITECTURES so it can use gcc8 on the gcc2hybrid

Reply:
Thanks.

Reply:
Not yet resolved.

Reply:
@trungnt2910 can you please add SECONDARY_ARCHITECTURES here? Otherwise we can't have a build for the gcc2hybrid

Reply:
How should I do this?
Just add a line?
Or do any other lines have to be changed

Reply:
Does using `SECONDARY_ARCHITECTURES` affect the building on non-x86 architectures (arm, ppc)?

Reply:
No those are "PRIMARY ARCHITECTURES" :)